commons/ava_testsuite/five_node_staking_gecko_network_tests.go

```diff
@@ -0,0 +1,73 @@
+package ava_testsuite
+
+import (
+	"github.com/kurtosis-tech/ava-e2e-tests/commons/ava_networks"
+	"github.com/kurtosis-tech/ava-e2e-tests/gecko_client"
+	"github.com/kurtosis-tech/kurtosis/commons/testsuite"
+	"github.com/palantir/stacktrace"
+	"github.com/sirupsen/logrus"
+	"time"
+)
+
+
+type FiveNodeStakingNetworkBasicTest struct{}
+func (test FiveNodeStakingNetworkBasicTest) Run(network interface{}, context testsuite.TestContext) {
+	castedNetwork := network.(ava_networks.NNodeGeckoNetwork)
+
+	// TODO check ALL nodes!
+	client, err := castedNetwork.GetGeckoClient(0)
+	if err != nil {
+	context.Fatal(stacktrace.Propagate(err, "Could not get client"))
+	}
+
+	peers, err := client.AdminApi().GetPeers()
+	if err != nil {
+	context.Fatal(stacktrace.Propagate(err, "Could not get peers"))
+	}
+
+	context.AssertTrue(len(peers) == 9)
```

Comment:
Also here???

Reply:
Actually this entire file's indentation is weird - are spaces and tabs being mixed?? Go's official guideline is tabs, so we should use that IMO

Reply:
no its not mixing of spaces and tabs, i did this manually on accident. fixed.